airbyte-integrations/connectors/source-coda/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_coda ./source_coda
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.version=1.0.0
```

Comment:
Why a major update and not a minor one?

Reply:
The docs schema had a field `owner_name` which has been renamed to `ownerName`.

Reply:
If I'm not mistaken, renaming a field is a major update.